File: Arrays/LargestElement.java
package Arrays;
import java.util.ArrayList;
import java.util.Scanner;
public class LargestElement {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n=sc.nextInt();
        ArrayList<Integer> arr= new ArrayList<Integer>();
        for(int i=0;i<n;i++)
            arr.add(sc.nextInt());
        int max= arr.get(0);
        for(Integer i : arr){
            if(max<i)
                max=i;
        }
        System.out.println("The Largest Element in Array is "+max);

    }
}

File: Arrays/LargestElement.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
    int n,a;
    cin>>n;
	vector<int>arr(n);
        for(int i=0;i<n;i++){
           cin>>a;
           arr.push_back(a);
        }
        int max= arr[0];
        for(auto i : arr){
            if(max<i)
                max=i;
        }
        cout<<"The Largest Element in Array is "<<max;
    }

